roles/install_python/README.md

```diff
@@ -43,11 +35,3 @@ This role will install Python on a managed host. To used this role, add it to th
   roles:
     - mirsg.install_python
 ```
-
-## License
-
-[BSD 3-Clause License](https://github.com/UCL-MIRSG/ansible-role-install-python/blob/main/LICENSE).
-
-## Author Information
-
-This role was created by the [Medical Imaging Research Software Group](https://www.ucl.ac.uk/advanced-research-computing/expertise/research-software-development/medical-imaging-research-software-group) at [UCL](https://www.ucl.ac.uk/).
```

Comment:
Why removing these bits?

Reply:
they should be in the top-level readme for the Collection instead of readmes for each role